lib/SPIRV/SPIRVInternal.h

```diff
@@ -943,6 +943,10 @@ std::string mangleBuiltin(StringRef UniqName, ArrayRef<Type *> ArgTypes,
 std::string getSPIRVFriendlyIRFunctionName(OCLExtOpKind ExtOpId,
                                            ArrayRef<Type *> ArgTys);
 
+/// Mangle a function in SPIR-V friendly IR manner
+std::string getSPIRVFriendlyIRFunctionName(const std::string &UniqName,
+                                           spv::Op OC, ArrayRef<Type *> ArgTys);
+
```

Comment:
Added description. I'm not sure if we can remove `UniqName` if we don't pass the instruction here, since full name can depend on decorations.
We can remove OpCode and get needed info based on `UniqName` like it is done in `OCLBuiltinFuncMangleInfo` but I prefer to avoid parsing info from strings which already available higher in call stack.